src/test/scala/com/amazon/deequ/profiles/ColumnProfilerTest.scala

```diff
@@ -400,4 +400,57 @@ class ColumnProfilerTest extends WordSpec with Matchers with SparkContextSpec
     }
   }
 
+  "return correct profile for the Titanic dataset" in withSparkSession { session =>
+    val data = session.read.format("csv")
+                .option("inferSchema", "true")
+                .option("header", "true")
+                .load("test-data/titanic.csv")
+    
+    val columnProfiles = ColumnProfiler.profile(data)
+
+    val idProfile = columnProfiles.profiles("PassengerId")
```

Comment:
you could refactor the code to check a column profile into its own method as its repeated many times

Reply:
Yes, makes sense.